scout/server/blueprints/mme/templates/mme/mme_submissions.html

```diff
@@ -0,0 +1,128 @@
+{% extends "layout.html" %}
+{% from "overview/institute_sidebar.html" import institute_actionbar %}
+
+{% block title %}
+{{ super() }} - Institutes
+{% endblock %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/dataTables.bootstrap5.min.css">
```

Comment:
They have updated that a bit, haven't they? Ah, actually, we have macros for including this stuff to keep it easy to bump:
https://github.com/Clinical-Genomics/scout/blob/4f7ac0a3c9fc8fb884e547312265966327b75643/scout/server/templates/utils.html#L236